doc/Dependencies.md

```diff
@@ -0,0 +1,20 @@
+Third Party Dependencies
+========================
+
+## Unaudited Required Dependencies
+
+`libtock-rs` has the following required dependencies, none of which are
```

Comment:
If we have that distinction (I think we should) I would add "build dependencies" here.

Reply:
Done.